eks-worker-al2.json

```diff
@@ -1,35 +1,51 @@
 {
   "variables": {
     "aws_region": "us-west-2",
-    "ami_name": "og-amazon-eks-node-{{timestamp}}",
-    "binary_bucket_region": "us-west-2",
+    "ami_name": null,
     "creator": "{{env `USER`}}",
-    "source_ami_owners": "137112412989",
-    "source_ami_filter_name": "amzn2-ami-minimal-hvm-*",
     "encrypted": "true",
+    "kms_key_id": "",
+    
     "aws_access_key_id": "{{env `AWS_ACCESS_KEY_ID`}}",
     "aws_secret_access_key": "{{env `AWS_SECRET_ACCESS_KEY`}}",
     "aws_session_token": "{{env `AWS_SESSION_TOKEN`}}",
 
-    "arch": "",
-    "instance_type": "",
-    "binary_bucket_name": "",
-    "docker_version": "",
+    "binary_bucket_name": "amazon-eks",
+    "binary_bucket_region": "us-west-2",
+    "kubernetes_version": null,
+    "kubernetes_build_date": null,
+    "docker_version": "18.09.9ce-2.amzn2",
```

Comment:
have it use the latest version instead `19.03.6ce-1.amzn2`

Reply:
bumped to 19.03.6ce-1.amzn2